site.yml

```diff
@@ -50,6 +50,7 @@ asciidoc:
     idseparator: '-'
     experimental: ''
     latest-version: 10.4
+    latest-download-version: 10.5.0
```

Comment:
@mmattel why did we set this to `10.5.0` ?
In 10.4 branch the "latest" version is `10.4.1`

Reply:
good point 🤦‍♂️

Reply:
I guess it was "accidental" when the backport was cherry-picked